etc/profile-m-z/reader.profile

```diff
@@ -0,0 +1,64 @@
+# Firejail profile for reader
+# Description: Better readability of web pages on the CLI
+# This file is overwritten after every install/update
+# Persistent local customizations
+include reader.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}/wayland-*
```

Comment:
Can we blacklist entire RUNUSER?

Reply:
Indeed we can. Thanks for the suggestion!